recipes/tarball.rb

```diff
@@ -24,17 +24,16 @@
 # Require for zk gem
 %w(patch gcc).each do |pkg|
   package pkg do
-    action :nothing
+    action :install
     only_if { node['solrcloud']['install_zk_gem'] }
-  end.run_action(:install)
+  end
 end
 
 chef_gem 'zk' do
-  action :nothing
+  action :install
   only_if { node['solrcloud']['install_zk_gem'] }
-end.run_action(:install)
```

Comment:
I believe this is carry over from when I was trying to figure out the apt-get issues. It works fine in chef12 but I think you are right about chef11. I'll undo this change.